src/services/isaacApi.ts

```diff
@@ -1,18 +1,28 @@
 import {siteSpecific} from "./site";
 
+export const LocalServer = siteSpecific(
+    "http://localhost:8080",
+    "http://localhost:8081",
```

Comment:
We use 8080 for both physics and Ada, and it would be difficult to modify this on the fly since it is hardcoded in our Jetty config. So I think this is wrong, and also does not need a `siteSpecific`.